packages/paste-website/src/pages/components/topbar/index.mdx

```diff
@@ -0,0 +1,164 @@
+import {Badge} from '@twilio-paste/badge';
+import {Box} from '@twilio-paste/box';
+import {Input} from '@twilio-paste/input';
+import {SearchIcon} from '@twilio-paste/icons/esm/SearchIcon';
+import {
+  ProductSwitcher,
+  ProductSwitcherButton,
+  ProductSwitcherItem,
+  useProductSwitcherState,
+} from '@twilio-paste/product-switcher';
+import {LogoTwilioIcon} from '@twilio-paste/icons/esm/LogoTwilioIcon';
+import {ProductSegmentIcon} from '@twilio-paste/icons/esm/ProductSegmentIcon';
+import {ProductFlexIcon} from '@twilio-paste/icons/esm/ProductFlexIcon';
+import {ProductEmailAPIIcon} from '@twilio-paste/icons/esm/ProductEmailAPIIcon';
+import packageJson from '@twilio-paste/topbar/package.json';
+import DefaultLayout from '../../../layouts/DefaultLayout';
+import {getFeature, getNavigationData} from '../../../utils/api';
+import {defaultTopbarExample, emptyTopbarExample} from '../../../component-examples/TopbarExamples.ts';
+
+export default DefaultLayout;
+
+export const getStaticProps = async () => {
+  const navigationData = await getNavigationData();
+  const feature = await getFeature('Topbar');
+  return {
+    props: {
+      data: {
+        ...packageJson,
+        ...feature,
+      },
+      navigationData,
+    },
+  };
+};
+
+<NormalizedComponentHeader
+  categoryRoute={SidebarCategoryRoutes.COMPONENTS}
+  githubUrl="https://github.com/twilio-labs/paste/tree/main/packages/paste-core/components/topbar"
+  storybookUrl="/?path=/story/components-topbar--"
+  data={props.data}
+/>
+
+---
+
+<contentwrapper>
+
+<PageAside data={mdxHeadings} />
+
+<content>
+
+<LivePreview
+  scope={{
+    Topbar,
+    Box,
+    Input,
+    Menu,
+    MenuItem,
+    useMenuState,
+    MenuBadge,
+    Badge,
+    ProductSwitcher,
+    ProductSwitcherButton,
+    ProductSwitcherItem,
+    useProductSwitcherState,
+    LogoTwilioIcon,
+    ProductSegmentIcon,
+    ProductFlexIcon,
+    ProductEmailAPIIcon,
+    SearchIcon,
+  }}
+  noInline
+>
+  {defaultTopbarExample}
+</LivePreview>
+
+## Guidelines
+
+### About Topbar
+
+A Topbar is sticky to the top of the screen and consists of application-wide controls including:
+
+- [Account Switcher](/components/account-switcher)
+- Search [Input](/components/input)
+- [User Dialog](#user-dialog)
+- [Product Switcher](/components/product-switcher)
+
+## Default Topbar
+
+Use a Topbar at the top of your application to house application-wide controls.
+
+Account-specific controls are aligned to the left, such as:
+
+- [Account Switcher](/components/account-switcher)
+- Account type indicators, such as subaccount [Badges](/components/badge)
+- Possible upgrade actions
+
+User-specific controls are aligned to the right, such as:
+
+- Search [Input](/components/input)
+- Support [Button](/components/button)
+- [Product Switcher](/components/product-switcher)
+- [User Dialog](#user-dialog)
+
+<LivePreview scope={{Topbar}} noInline>
+  {emptyTopbarExample}
+</LivePreview>
+
+## User Dialog
+
+Coming soon!
```

Comment:
Is User Dialog still going on this page?

Reply:
no

Reply:
it's its own package